package/samplers/value_at_risk/sampler.py

```diff
@@ -106,6 +107,19 @@ class RobustGPSampler(BaseSampler):
             The list of parameters determined externally rather than being decision variables.
             For these parameters, `suggest_float` samples random values instead of searching
             values that optimize the objective function.
+        noisy_suggestion:
+            If this option is enabled, suggested values will not be the nominal value but the
+            nominal value plus added noise and clipping. This option is useful when a user want
+            to evaluate the objective function against values with added noise rather than
+            nominal values. If this option is enabled, nominal values can be retrieved using
+            `get_nominal_params`.
+        nominal_ranges:
```

Comment:
[Question] I believe `uniform_input_noise_rads` and `nominal_range` serve overlapping purposes. For instance, the code:
```python
uniform_input_noise_rads={"x": 0.1}
...
x = trial.suggest_float("x", 5, 10)
```
may be essentially the same as:
```python
nominal_ranges={"x": [5, 10]}
...
x = trial.suggest_float("x", 4.9, 10.1)
```
from the user's perspective.

If so, we may be able to use `uniform_input_noise_rads` instead of introducing a new `nominal_ranges`.